go.mod

```
@@ -70,7 +70,7 @@ require (
 	github.com/mitchellh/hashstructure/v2 v2.0.2
 	github.com/moby/sys/mountinfo v0.6.2
 	github.com/np-guard/cluster-topology-analyzer v1.7.0
-	github.com/np-guard/netpol-analyzer v0.2.2
+	github.com/np-guard/netpol-analyzer v0.3.2
```

- https://github.com/stackrox/stackrox/pull/7316
 
 ```suggestion
	github.com/np-guard/netpol-analyzer v0.4.0
```

---

I see that Dependabot thinks that `0.y+1` is compatible with `0.y`. This would be true if that is `1.y`, but there is no compatibility guarantee between the `0.y` versions. https://semver.org/#spec-item-4

Can we somehow configure it to respect that?

---

This PR  should remain with `v0.3.2` of `netpol-analyzer`.
We have a new release of `v0.4.2` for which we will open soon a separate PR (on top of this PR/branch), since it adds support for the `connectivity-diff` command.